addons/binding/org.openhab.binding.onebusaway/ESH-INF/thing/api.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="onebusaway"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <bridge-type id="api">
+        <label>OneBusAway API</label>
```

Comment:
No, this is not the "API key" here, this is the "bridge" (which needs an API key as a configuration). So the "Thing" that requires the key is the web service - that's why I was asking for a more speaking name.